src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -63,14 +64,24 @@ def load_arguments(self, _):
         c.argument('top', arg_type=top_arg_type)
         c.argument('all_', options_list=['--all'], action='store_true', help="List all items.")
         c.argument('fields', arg_type=fields_arg_type)
+        c.argument('sku', help='The sku of App Configuration', arg_type=get_enum_type(['free', 'standard']))
 
     with self.argument_context('appconfig create') as c:
         c.argument('location', options_list=['--location', '-l'], arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
-        c.ignore('sku')
+        c.argument('assign_identity', nargs='*', validator=validate_assigned_identity, arg_group='Managed Service Identity', is_preview=True, help="Accept system or user assigned identities separated by spaces. Use '[system]' to refer system assigned identity, or a resource id to refer user assigned identity. Check out help for more examples")
```

Comment:
>arg_group='Managed Service Identity' [](start = 87, length = 36)

Are we planning to add more arguments like role and scope to this group in future? If not, does it make sense to not have a separate arg_group for this?

Reply:
We may keep adding arguments for store creation, but not for scoping identities. This should be done in "az role". The assign_identity means we want to create a system assigned or associate a user assigned identity to this configuration store. The scope of the identity won't be modified here. 

---
In reply to: [364511654](https://github.com/Azure/azure-cli/pull/11795#discussion_r364511654) [](ancestors = 364511654)

Reply:
Got it. In that case, should we put this single argument in a separate arg_group?

---
In reply to: [364520390](https://github.com/Azure/azure-cli/pull/11795#discussion_r364520390) [](ancestors = 364520390,364511654)

Reply:
We don't actually need the arg_group. 

---
In reply to: [364525333](https://github.com/Azure/azure-cli/pull/11795#discussion_r364525333) [](ancestors = 364525333,364520390,364511654)